pylxd/container.py

```diff
@@ -65,7 +66,7 @@ def create(cls, client, config, wait=False):
         response = client.api.containers.post(json=config)
 
         if response.status_code != 202:
-            raise RuntimeError('Error creating instance')
+            raise exceptions.CreateFailed()
```

Comment:
I don't really want to use the whole response, though I agree we need more information. The response json's message param would seem appropriate as a good start. I'll add it.